lua/mapvote/server/modules/rtv.lua

```diff
@@ -128,11 +128,11 @@ function RTV.CanVote( ply )
     local conf = MapVote.GetConfig()
 
     if conf.RTVWait >= CurTime() then
-        return false, "You must wait a bit before voting!"
+        return false, "You must wait " .. string.NiceTime( conf.RTVWait - CurTime() ) .. " before voting for a new map again!"
```

Comment:
I dont like the word `again` here, this is really the first vote on the map and it might be this persons first time voting. Also want to differentiate it from the other message